src/components/activity/screens/ActivityList/actions/index.js

```diff
@@ -34,29 +34,52 @@ export function getActivityList() {
 
 export function getActivityListFromRequest(request) {
   if (request.length > 0) {
-    return (dispatch, getState) => executeSql(
-      `SELECT id, title, description, activity_date, content_source, type, interlocutor_id
+    requestList = request.split(' ');
+    return (dispatch, getState) => {
+      let promises = [];
+      request = `SELECT id, title, description, activity_date, content_source, type, interlocutor_id
```

Comment:
Same remark as in src/components/interlocutor/screens/InterlocutorList/actions/index.js : reusing function parameter

Reply:
requestSql